src/sentry/models/projectcodeowners.py

```diff
@@ -16,8 +16,6 @@ class ProjectCodeOwners(DefaultFieldsModel):
     __core__ = False
     # no db constraint to prevent locks on the Project table
     project = FlexibleForeignKey("sentry.Project", db_constraint=False)
-    # projectcodeowners should work without a repo-based integration.
-    organization_integration = FlexibleForeignKey("sentry.OrganizationIntegration", null=True)
```

Comment:
We need to update `ProjectCodeOwnersSerializer` as well right? Since we use this field there?

Reply:
@wedamija good catch!! yes, I just updated it.